tests/integration/targets/iso_customize/tasks/iso_customize.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: "Customize ISO file"
+  community.general.iso_customize:
+    src_iso: "{{ test_dir }}/test.iso"
+    dest_iso: "{{ test_dir }}/{{ dest_iso_name }}"
+    delete_files:
+      - "/test01.cfg"
+    add_files:
+      - src_file: "{{ test_dir }}/test01.cfg"
+        dest_file: "/preseed/ubuntu.seed"
+  register: customize_iso_result
+
+- name: "Display the ISO customized result"
+  debug: var=customize_iso_result
+
+- name: Test customize iso result
+  assert:
+    that:
+    - customize_iso_result is not failed
+
+- debug: var=ansible_distribution
+
+# Only need to check the files is deleted/added as the requested modifications 
+# in one type of OS(take MacOS here)
+- block:
+    - name: Create sub directory to mount customized ISO
+      file:
+        path: '{{ test_dir }}/iso_mount'
+        state: directory
+
```

Comment:
Would this be better off at `prepare.yml` ?

Reply:
Thanks. Fix it